### PR TITLE
m365certupdates to permissiongrantpolicy 

### PR DESCRIPTION
**Pull request by arpitha-dhanapathi**
PR:9114

### CODE REVIEW

**Review by psignoret — DISMISSED** [review]
Looks good! I think the only thing that's missing is the changelog update and a couple minor suggestions below.

**Comment by arpitha-dhanapathi** [comment]
@psignoret  - could you please review and signoff? Thanks.

**Review by arpitha-dhanapathi — COMMENTED** [review]
Made the changes

**Review by arpitha-dhanapathi — COMMENTED** [review]
reviewed and submitted changes

**Review by arpitha-dhanapathi — COMMENTED** [review]
reviewed and updated

File: api-reference/beta/resources/permissiongrantconditionset.md [hunk 1]
```diff
@@ -49,6 +50,7 @@ A permission grant condition set contains several conditions. For an event to ma
     "clientApplicationIds": [ "string" ],
     "clientApplicationTenantIds": [ "string" ],
     "clientApplicationPublisherIds": [ "string" ],
-    "clientApplicationsFromVerifiedPublisherOnly": false
+    "clientApplicationsFromVerifiedPublisherOnly": false,
```
Comment (JarbasHorst) [inline api-reference/beta/resources/permissiongrantconditionset.md:53]: For consistency, the JSON representation should not contain examples. Please, replace "false" in both cases with "Boolean".

File: api-reference/v1.0/api/permissiongrantpolicy-list-excludes.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ If successful, this method returns a `200 OK` response code and a collection of
 
 ### Request
 
-The following is an example of the request to retrieve the **excludes** condition sets of the built-on permission grant policy `microsoft-application-admin`. This permission grant policy includes all delegated permissions, and all application permissions excluding application permissions for Microsoft Graph and application permissions for Azure AD Graph.
+The following is an example of the request to retrieve the **excludes** condition sets of the built-in permission grant policy `microsoft-application-admin`. This permission grant policy includes all delegated permissions, and all application permissions excluding application permissions for Microsoft Graph and application permissions for Azure AD Graph.
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/permissiongrantpolicy-list-excludes.md:55]: Before using acronyms, I recommend you to spell out the words first followed by the acronym in parentheses. In this case, you don't need an acronym, so I recommend you to only spell out AD. Suggested revision:

- ...permissions for Azure Active Directory Graph.

File: changelog/Microsoft.DirectoryServices.json [hunk 3]
```diff
@@ -1,5 +1,41 @@
 {
   "changelog": [
+    {
+     "ChangeList": [
+      {
```
Comment (JarbasHorst) [inline changelog/Microsoft.DirectoryServices.json:5]: Your PR doesn't include this change. Please remove this entire changelog record.

File: changelog/Microsoft.DirectoryServices.json [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
+      "SubArea": "Directory management"
+    },
+    {
+    "ChangeList": [
+        {
+          "Id": "8c11b65a-06ba-4eb6-b022-5b3363341607",
+          "ApiChange": "Property",
+          "ChangedApiName": "certifiedClientApplicationsOnly",
+          "ChangeType": "Addition",
+          "Description": "Added the **certifiedClientApplicationsOnly** property to [permissionGrantConditionSet](https://docs.microsoft.com/en-us/graph/api/resources/permissionGrantConditionSet?view=graph-rest-beta) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.DirectoryServices.json:28]: Please, add "the" before "[permissionGrantConditionSet]".

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/permissiongrantpolicy-post-includes.md [hunk 5]
```diff
@@ -50,12 +50,14 @@ If successful, this method returns a `201 Created` response code and an [permiss
...
+### Example 1: Create a permission grant policy for client apps that are from verified pubilshers 
+
 ### Request
 
-In this example, *all* delegated permissions for client apps from verified publishers are included in the permission grant policy. Because all the other conditions from the [permissionGrantConditionSet](../resources/permissiongrantconditionset.md) were omitted, they will take their default values, which in each case is the most-inclusive.
+In this example, *all* delegated permissions for client apps that are from verified publishers are included in the permission grant policy. Because all the other conditions from the [permissionGrantConditionSet](../resources/permissiongrantconditionset.md) were omitted, they will take their default values, which in each case is the most-inclusive.
 
 
-# [HTTP](#tab/http)
+### [HTTP](#tab/http)
```
[the remark this answers is not in the document]
Comment (arpitha-dhanapathi) [reply api-reference/beta/api/permissiongrantpolicy-post-includes.md:60]: the tabs (eg., (#tab/http)) weren't added separately by me. They were in the docs for the first example, which I copied over to create the second example. Should all of these be still removed?

File: changelog/Microsoft.DirectoryServices.json (hunk 3 above)
Comment (JarbasHorst) [reply changelog/Microsoft.DirectoryServices.json:5]: After some investigation with @arpitha-dhanapathi, we figured out that her PR introduces a JSON code that is well-formatted. GitHub understands this as a change and displays this changelog record as an addition.

For comparison, here the same changelog record from the main branch:


MicrosoftTeams-image (21)

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

File: api-reference/beta/api/permissiongrantpolicy-post-includes.md [hunk 6]
```diff
@@ -90,7 +92,78 @@ Content-Type: application/json
...
+
+# [HTTP](#tab/http)
+<!-- {
+  "blockType": "request",
+  "truncated": true,
+  "name": "permissiongrantpolicy_create_includes"
+}-->
+
+```http
+POST https://graph.microsoft.com/beta/policies/permissionGrantPolicies/{id}/includes
```
Comment (JarbasHorst) [inline api-reference/beta/api/permissiongrantpolicy-post-includes.md:139]: Please, replace the {id} placeholder with a real example. Please, apply this change also to HTTP request in Example 1.

File: api-reference/beta/api/permissiongrantpolicy-list-excludes.md [hunk 7]
```diff
@@ -54,7 +54,7 @@ If successful, this method returns a `200 OK` response code and a collection of
 
 ### Request
 
-The following is an example of the request to retrieve the **excludes** condition sets of the built-on permission grant policy `microsoft-application-admin`. This permission grant policy includes all delegated permissions, and all application permissions excluding application permissions for Microsoft Graph and application permissions for Azure AD Graph.
+The following is an example of the request to retrieve the **excludes** condition sets of the built-in permission grant policy `microsoft-application-admin`. This permission grant policy includes all delegated permissions, and all application permissions excluding application permissions for Microsoft Graph and application permissions for Azure Active Direcotry Graph.
```
Comment (JarbasHorst) [inline api-reference/beta/api/permissiongrantpolicy-list-excludes.md:57]: Please, replace "Direcotry" with "Directory".

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Review by arpitha-dhanapathi — COMMENTED** [review]
updated with all suggestions

**Review by arpitha-dhanapathi — COMMENTED** [review]
Reviewed and updated

**Comment by arpitha-dhanapathi** [comment]
@Lauragra  - content review is complete by Jarba. Should I mark it as 'ready to merge'?

**Comment by Lauragra** [comment]
> @Lauragra - content review is complete by Jarba. Should I mark it as 'ready to merge'?

Yes, please set ready to merge when the schema is released and the APIs are available to users.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**